test/configuration_test.rb

```diff
@@ -201,7 +201,7 @@
       @config.destination.must_equal Pathname.new(__dir__)
       assert @config.sources == [__dir__ + '/fixtures/javascripts'],
         "Expected #{ @config.sources } to eq [#{ __dir__ }/fixtures/javascripts'], found: #{ @config.sources.inspect }"
-      @config.__send__(:types).types.must_equal [:javascript, :stylesheet, :movie]
+      @config.__send__(:types).types.must_equal [:javascript, :stylesheet, :image, :movie]
```

Comment:
I am not totally sure of what this test checks, but it was failing unless I added `:image`. Is this correct behavior?